scripts/deploy_teleporter.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e
+
+TELEPORTER_PATH=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")"
+    cd .. && pwd
+)
+
+if ! command -v forge &> /dev/null; then
+    echo "forge not found. You can install by calling $TELEPORTER_PATH/scripts/install_foundry.sh" && exit 1
```

Comment:
nit : in `abi_bindings.sh` we call the script to install if not installed already, do we want to do the same? 

If not, can we change wording to just "forge not found, install by calling ..."

Reply:
Yep, changed.